examples/fastify-ssr/package.json

```diff
@@ -29,9 +28,11 @@
     "graphql-hooks-memcache": "^2.2.1",
     "graphql-hooks-ssr": "^2.2.1",
     "isomorphic-unfetch": "^3.0.0",
+    "prop-types": "^15.7.2",
     "react": "^17.0.0",
     "react-dom": "^17.0.0",
-    "react-tree-walker": "^4.3.0"
```

Comment:
This also has peer [dependency of React < 17](https://github.com/ctrlplusb/react-tree-walker/blob/master/package.json#L36), but it's also unused in this sample project. That's the reason I removed it.